src/GraphControl/Control/Grapher.cpp

```diff
@@ -199,6 +199,8 @@ namespace GraphControl
         {
             if (auto analyzer = graph->GetAnalyzer())
             {
+                graph->GetOptions().SetAllowKeyGraphFeaturesForFunctionsWithParameters(true);
```

Comment:
I think we should consider replacing this with a call to UpdateGraphOptions, that way we can keep this graph creation consistent with the regular graph creation.